code/game/gamemodes/changeling/powers/headcrab.dm

```diff
@@ -6,6 +6,10 @@
 	dna_cost = 1
 	req_human = 1
 
+/obj/effect/proc_holder/changeling/headcrab/try_to_sting(mob/user, mob/target)
+    if(alert("Are you sure you wish to do this? This action cannot be undone.",,"Yes","No")=="Yes")
```

Comment:
Does this skip the `can_sting` check?

Might be a good idea to do:
```
if(alert("Are you sure you wish to do this? This action cannot be undone.",,"Yes","No") == "No")
    return
..()
```
if it does.

Reply:
Most recent change actually completely broke it, pressing yes does nothing. I have no clue what's gone wrong really.